test/e2e/azure_test.go

```diff
@@ -388,17 +388,17 @@ var _ = Describe("Workload cluster creation", func() {
 				})
 			})
 
-			By("Running a security scanner", func() {
-				KubescapeSpec(ctx, func() KubescapeSpecInput {
-					return KubescapeSpecInput{
-						BootstrapClusterProxy: bootstrapClusterProxy,
-						Namespace:             namespace,
-						ClusterName:           clusterName,
-						FailThreshold:         e2eConfig.GetVariable(SecurityScanFailThreshold),
-						Container:             e2eConfig.GetVariable(SecurityScanContainer),
-						SkipCleanup:           skipCleanup,
-					}
-				})
+			By("Running a security scanner -- skipping because it's broken", func() {
```

Comment:
Let's just remove this entire `By` block. git history (and the `KubescapeSpec` func) will allow us to easily retrieve it.

Reply:
I was gonna say let's just merge this as-is so we can unblock all the PRs in the queue but we have an Azure bastion flake in this test run so yeah, go ahead and delete this whole test section from the VMSS test flow and let's try again!